### DISCUSSION TITLE
Returning HTML/CSS efficiently

### DISCUSSION THREAD

**Discussion by rthrfrd** [uNetworking/uWebSockets#1688] [Show and tell]
First of all, thank you very much for this library! I've been using it with coroutines and it's been lovely.

I didn't find a good solution with similar efficiency for producing dynamic & complex HTML pages to go along with this, so I wanted to make and share my own contribution in case anyone finds it useful. Hopefully it makes avoiding copies and getting to the first byte as easy as possible, but any critique/feedback to improve/enhance it is much appreciated!

https://github.com/rthrfrd/webxx

Thanks again,

Alex 👋

**Comment by uNetworkingAB** [comment]
You can make an example like "What's my user agent" and we can benchmark it
